### PR TITLE
Bugfix optional TLS settings

### PR DESCRIPTION
**Pull request by JeroenBo**
##### SUMMARY
When tls_connect and tls_accept are not specified in the playbook it defaults to 1 which removes the current TLS/PSK settings of the host in Zabbix.

This update will remove the default value of 1 and only updates the data when needed.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
zabbix_host.py

##### ADDITIONAL INFORMATION
When tls_connect and tls_accept are not specified in the playbook it defaults to 1 which removes the current TLS/PSK settings of the host in Zabbix

### CODE REVIEW

**Comment by mu1f407** [comment]
Hi @JeroenBo, thank you for PR! There are some failing checks though. Could you please take a look at it? Some linter checks are failing (trailing whitespace, missing space after comma,...) and module documentation should be adjusted for changed parameter default. Also please include changelog fragment (see [CONTRIBUTING](https://github.com/ansible-collections/community.zabbix/blob/main/CONTRIBUTING.md)). Thanks!

If you need any help, just let me know.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -22,6 +22,7 @@ Bugfixes
 - zabbix_action - will no longer wipe `esc_step_to` and `esc_step_from` (https://github.com/ansible-collections/community.zabbix/issues/692)
 - zabbix_agent - added support for zabbix-agent on Ubuntu 22.04 (https://github.com/ansible-collections/community.zabbix/pull/681)
 - zabbix_agent - now properly creates webroot for issuing LE certificates (https://github.com/ansible-collections/community.zabbix/pull/677, https://github.com/ansible-collections/community.zabbix/pull/682)
+- zabbix-agent - will no longer wipe tls_connect en tls_accept settings when not specified on update (https://github.com/ansible-collections/community.zabbix/pull/753)
```
Comment (mu1f407) [inline CHANGELOG.rst:25]: Changelog shouldn't go directly into the main CHANGELOG file. Please add fragment into /changelogs/fragments directory. See existing files as an examples. This should be categorized as minor_changes.
```suggestion
---
minor_changes:
  - zabbix_host - will no longer wipe tls_connect en tls_accept settings when not specified on update
```
Above should go into /changelogs/fragments/753-host-tls-settings.yml for example.

**Review by mu1f407 — CHANGES_REQUESTED** [review]
@JeroenBo After requested changes I am happy to approve your PR. Thanks!

**Review by D3DeFi — APPROVED** [review]
Looks good! Thank you :)

@mu1f407 your requested changes were applied.

**Review by mu1f407 — APPROVED** [review]

**Comment by mu1f407** [comment]
@JeroenBo Thank you for your contribution! Highly appreciated.

**Pull request merged**